web/pdf_document_properties.js

```diff
@@ -13,7 +13,12 @@
  * limitations under the License.
  */
 
-import { cloneObj, getPDFFileNameFromURL, NullL10n } from './ui_utils';
+import {
```

Comment:
Can we put these behind each other like we do in all other places (for example https://github.com/mozilla/pdf.js/pull/9586/files#diff-c7f5d75f7b6b0f13392f9d4c49a0c990L17)? It's just to keep a consistent style.